editor/plugins/game_view_plugin.cpp

```diff
@@ -344,11 +344,17 @@ void GameView::_embed_options_menu_menu_id_pressed(int p_id) {
 	switch (p_id) {
 		case EMBED_RUN_GAME_EMBEDDED: {
 			embed_on_play = !embed_on_play;
-			EditorSettings::get_singleton()->set_project_metadata("game_view", "embed_on_play", embed_on_play);
+			int game_mode = EDITOR_GET("run/window_placement/game_embed_mode");
+			if (game_mode == 0) { // Do not save if overridden by editor.
```

Comment:
```suggestion
			if (game_mode == 0) { // Save only if not overridden by editor.
```
Better matches the condition.
Same below.